google/data_source_dns_managed_zone.go

```diff
@@ -9,12 +15,12 @@ func dataSourceDnsManagedZone() *schema.Resource {
 		Schema: map[string]*schema.Schema{
 			"dns_name": &schema.Schema{
 				Type:     schema.TypeString,
-				Computed: true,
```

Comment:
I think this should still be computed as Computed values are still Optional, but DNS name is probably a required value for a managed zone. Even though we set the value to the GCP resource's value right now, Optional can potentially cause this value to not be set.